pvlib/temperature.py

```diff
@@ -443,6 +444,50 @@ def faiman(poa_global, temp_air, wind_speed=1.0, u0=25.0, u1=6.84):
     return temp_air + temp_difference
 
 
+def ross(poa_global, temp_air, noct):
+    r'''
+    Calculate cell temperature using the Ross model.
+
+    The Ross model [1]_ assumes that difference betwen cell temperature
+    and ambient temperature is proportional to the plane of array irradiance,
+    and ignores the effects of wind.
```

Comment:
@adriesse recently reminded us of #433. In that spirit, I suggest we encourage new functions to include documentation about any additional assumptions that went into the model development e.g. hourly data or zenith restrictions.

Reply:
Adding the implicit assumption of steady conditions.